tools/mmp/driver.cs

```diff
@@ -1070,7 +1070,7 @@ static void HandleFramework (StringBuilder args, string framework, bool weak)
 			else
 				BuildTarget.Frameworks.Add (name);
 
-			if (!framework.EndsWith (".framework", StringComparison.Ordinal))
+			if (!framework.EndsWith (".framework", StringComparison.OrdinalIgnoreCase))
```

Comment:
That seems incomplete
```
git grep "EndsWith (\".framework\""
```
returns many other cases (some shared my mtouch) where we check for path ending with `.framework`

Reply:
So the other one directly in mmp does:

```
if (v.EndsWith (".framework", true, CultureInfo.InvariantCulture))
```

with true being ignoreCase.

You are correct that mtouch doesn't have this fix, I could look at doing it in this PR, but I wouldn't want to write a test given the existing MSBuild testing situation.

Reply:
I did not look if the code itself was used but the `common/*` files are also used by `mmp`

Reply:
I don't believe the common bits are needed to get the macOS use case working (given my test).

I don't want to change the ios\common bits without a test.